### DISCUSSION TITLE
Destroy items

### DISCUSSION THREAD

**Discussion by liexes** [diamond-rip/Eden#404] [Ideas]
Add the fuction to break blocks per arena, the blocks allow place in "Extra items" on the kit editor, and the blocks places in this option is a blocks to allow break in this arena. Sorry for my english, i am speaking spanish.

**Comment by liexes** [comment]
@GoodestEnglish

**Reply by GoodestEnglish** [reply]
What gamemode do you need for this feature?
Adding this feature will be pretty confusing, as kit and arena can be mixed together.

**Reply by liexes** [reply]
For cavefight is a own mode of my server

**Reply by GoodestEnglish** [reply]
I can add this feature, but it will be according to kits, not arenas. Does that still alright?

**Reply by liexes** [reply]
If I mean, I want the kit editor in the extra materials section to put the materials that you allow to destroy in that kit, or you can add it as you see fit.

**Reply by GoodestEnglish** [reply]
Blocks which placed by a player can be destroyed by player already. 
But I got what you mean. I can see what I can add. But in a low priority

**Reply by liexes** [reply]
Thx, please add need this
